addons/mail/static/src/components/rtc_call_viewer/rtc_call_viewer.xml

```diff
@@ -51,15 +51,15 @@
 
             <!-- Dialogs -->
             <t t-if="messaging.userSetting.rtcConfigurationMenu.isOpen">
-                <Dialog size="'small'" title="'Settings'" t-on-dialog-closed="rtcCallViewer.onRtcSettingsDialogClosed">
+                <Dialog size="'small'" title="'Settings'" onClosed="rtcCallViewer.onRtcSettingsDialogClosed">
```

Comment:
Also title not translatable (need to make a new task to fix them)

Reply:
I'll make another PR for them

Reply:
Task: https://www.odoo.com/web#id=2703292&menu_id=4720&cids=1&action=333&active_id=1519&model=project.task&view_type=form
PR: https://github.com/odoo/odoo/pull/80673